spec/acceptance/callbacks_spec.rb

```diff
@@ -138,9 +138,9 @@ def name
       end
     end
 
-    FactoryBot.register_strategy(:custom_before, custom_before)
-    FactoryBot.register_strategy(:custom_after, custom_after)
-    FactoryBot.register_strategy(:totally_custom, totally_custom)
+    FactoryBot::Internal.register_strategy(:custom_before, custom_before)
```

Comment:
`FactoryBot.register_strategy` is still around as a non-internal method, so I think we can leave a lot of these specs as they were.